`test_plot_composition` test failures when expecting a string but get an object description

Building for nixpkgs on aarch64-darwin (MacOS), I see the following test errors:

```sh
=================================== FAILURES ===================================
___________________________ test_complex_composition ___________________________

    def test_complex_composition():
        p1 = plot.red
        p2 = plot.green + g.points + rotate.plot_title + legend.bottom
        p3 = plot.blue + g.cols + legend.left + tag("b)")
        p4 = plot.yellow + g.points + rotate.axis_title_y
        p5 = plot.cyan + tag("c)") + theme(plot_tag=element_text(margin={"t": 10}))
        p6 = (
            plot.orange
            + rotate.axis_title_y
            + tag("o)", "topright")
            + theme(plot_tag=element_text(va="bottom"))
        )
        p = p1 | p2 | p3 / p4 / (p5 | p6)
>       assert p == "complex_composition"
E       AssertionError: assert Beside(items=[<plotnine.ggplot.ggplot object at 0x33210a390>, <plotnine.ggplot.ggplot object at 0x3321099d0>, Stack(it...50>, Beside(items=[<plotnine.ggplot.ggplot object at 0x33210a8d0>, <plotnine.ggplot.ggplot object at 0x33210b710>])])]) == 'complex_composition'

tests/test_plot_composition.py:117: AssertionError
______________________________ test_and_operator _______________________________

    def test_and_operator():
        p1 = plot.red
        p2 = plot.green
        p3 = plot.blue
        p4 = plot.brown
        p = (p1 | p2 | (p3 / p4)) & theme_gray()
>       assert p == "and_operator"
E       AssertionError: assert Beside(items=[<plotnine.ggplot.ggplot object at 0x33213a450>, <plotnine.ggplot.ggplot object at 0x332139790>, Stack(items=[<plotnine.ggplot.ggplot object at 0x33213a750>, <plotnine.ggplot.ggplot object at 0x332139a90>])]) == 'and_operator'

tests/test_plot_composition.py:135: AssertionError
______________________________ test_mul_operator _______________________________

    def test_mul_operator():
        p1 = plot.red
        p2 = plot.green
        p3 = plot.blue
        p4 = plot.brown
        p = (p1 | p2 | (p3 / p4)) * theme_gray()
>       assert p == "mul_operator"
E       AssertionError: assert Beside(items=[<plotnine.ggplot.ggplot object at 0x137c1a5d0>, <plotnine.ggplot.ggplot object at 0x33211a990>, Stack(items=[<plotnine.ggplot.ggplot object at 0x33215f1d0>, <plotnine.ggplot.ggplot object at 0x137c19f10>])]) == 'mul_operator'

tests/test_plot_composition.py:144: AssertionError
______________________________ test_plus_operator ______________________________

    def test_plus_operator():
        p1 = plot.red
        p2 = plot.green
        p3 = plot.blue
        p4 = plot.brown
        p = (p1 | p2 | (p3 / p4)) + theme_gray()
>       assert p == "plus_operator"
E       AssertionError: assert Beside(items=[<plotnine.ggplot.ggplot object at 0x33215fad0>, <plotnine.ggplot.ggplot object at 0x33215fb90>, Stack(items=[<plotnine.ggplot.ggplot object at 0x33211a390>, <plotnine.ggplot.ggplot object at 0x332118350>])]) == 'plus_operator'

tests/test_plot_composition.py:153: AssertionError
=============================== warnings summary ===============================
tests/test_layers.py::TestRasterizing::test_geom_point
tests/test_layers.py::TestRasterizing::test_geom_path
  /nix/store/8y5p9wwgr5kl59impkj1jknmrbacpsn3-python3.13-matplotlib-3.10.3/lib/python3.13/site-packages/matplotlib/backends/backend_pdf.py:1761: DeprecationWarning: 'mode' parameter is deprecated and will be removed in Pillow 13 (2026-10-15)

-- Docs: https://docs.pytest.org/en/stable/how-to/capture-warnings.html
=========================== short test summary info ============================
FAILED tests/test_plot_composition.py::test_complex_composition - AssertionError: assert Beside(items=[<plotnine.ggplot.ggplot object at 0x33...
FAILED tests/test_plot_composition.py::test_and_operator - AssertionError: assert Beside(items=[<plotnine.ggplot.ggplot object at 0x33...
FAILED tests/test_plot_composition.py::test_mul_operator - AssertionError: assert Beside(items=[<plotnine.ggplot.ggplot object at 0x13...
FAILED tests/test_plot_composition.py::test_plus_operator - AssertionError: assert Beside(items=[<plotnine.ggplot.ggplot object at 0x33...
====== 4 failed, 133 passed, 1 skipped, 8 deselected, 2 warnings in 8.14s ======
error: builder for '/nix/store/mpmavslwvp0dq8qpdxxw937mzcqc8hnr-python3.13-plotnine-0.15.0.drv' failed with exit code 1;
       last 25 log lines:
       > tests/test_plot_composition.py:144: AssertionError
       > ______________________________ test_plus_operator ______________________________
       >
       >     def test_plus_operator():
       >         p1 = plot.red
       >         p2 = plot.green
       >         p3 = plot.blue
       >         p4 = plot.brown
       >         p = (p1 | p2 | (p3 / p4)) + theme_gray()
       > >       assert p == "plus_operator"
       > E       AssertionError: assert Beside(items=[<plotnine.ggplot.ggplot object at 0x33215fad0>, <plotnine.ggplot.ggplot object at 0x33215fb90>, Stack(items=[<plotnine.ggplot.ggplot object at 0x33211a390>, <plotnine.ggplot.ggplot object at 0x332118350>])]) == 'plus_operator'
       >
       > tests/test_plot_composition.py:153: AssertionError
```

Possibly a `toString` issue?

--- a/tests/conftest.py
+++ b/tests/conftest.py
@@ -127,7 +127,11 @@ ggplot.build_test = build_test
 
 
 def pytest_assertrepr_compare(op, left, right):
-    if isinstance(left, ggplot) and isinstance(right, str) and op == "==":
+    if (
+        isinstance(left, (ggplot, Compose))
+        and isinstance(right, str)
+        and op == "=="
+    ):
         msg = (
             "images not close: {actual:s} vs. {expected:s} "
             "(RMS {rms:.2f})".format(**left._err)
@@ -252,10 +256,10 @@ def composition_equals(cmp: Compose, name: str) -> bool:
     test_file = inspect.stack()[1][1]
     filenames = make_test_image_filenames(name, test_file)
 
-    cmp = cmp + theme(dpi=DPI)
+    _cmp = cmp + theme(dpi=DPI)
 
     with _test_cleanup():
-        cmp.save(filenames.result)
+        _cmp.save(filenames.result)
 
     if filenames.baseline.exists():
         shutil.copyfile(filenames.baseline, filenames.expected)

